libs/sdk-ui-filters/src/RankingFilter/RankingFilterDropdownBody.tsx

```diff
@@ -0,0 +1,113 @@
+// (C) 2020 GoodData Corporation
+import React, { useState, useCallback } from "react";
+import Button from "@gooddata/goodstrap/lib/Button/Button";
+import { IRankingFilter, newRankingFilter, ObjRefInScope } from "@gooddata/sdk-model";
+import { WrappedComponentProps, injectIntl, FormattedMessage } from "react-intl";
+import { isEqual, xorWith, isEmpty } from "lodash";
```

Comment:
Please use individual imports of lodash (like `import isEqual from "lodash/isEqual";`), we do not have a mechanism in place to fix them.